src/main/resources/feedbackQuestionConstSumSubmissionFormTemplate.html

```diff
@@ -1,16 +1,16 @@
-<div>
+<div class="col-sm-6">
   ${constSumSubmissionFormOptionFragments}
-  <div id="constSumInfo-${questionIndex}-${responseIndex}" class="form-group" ${constSumOptionVisibility}>
-    <div class="col-sm-12">
-      <hr class="margin-top-0">
-      <p class="text-color-blue" id="constSumMessage-${questionIndex}-${responseIndex}">
-      </p>
-      <hr>
-    </div>
+</div>
+<div id="constSumInfo-${questionIndex}-${responseIndex}" class="form-group col-sm-9" ${constSumOptionVisibility}>
+  <div class="col-sm-12 text-center">
```

Comment:
Should be `col-sm-6` right? Also, the `col-sm-9` for parent `div` doesn't add much value so you can remove it.

This is what `col-sm-6` give you.

![image](https://user-images.githubusercontent.com/13977235/35192551-b044d0aa-fecf-11e7-8a7d-998e4678b6c3.png)

![image](https://user-images.githubusercontent.com/13977235/35192606-eec42e74-fed0-11e7-93a5-f3702ccc7ca6.png)